packages/packages/api/src/index.ts

```diff
@@ -23,6 +23,10 @@ export {
 export {
   DocumentList,
   documentListSchema,
+  DocumentQueryProgress,
+  documentQueryProgress,
+  DocumentQuery,
+  documentQuerySchema,
```

Comment:
@Orta21 we have changes on the SDK but no changes on the package.json and lock. This is missing, right?

Reply:
Weird it shouldve been updated yea